context/app/static/js/components/workspaces/WorkspacesList.jsx

```diff
@@ -30,17 +44,18 @@ function WorkspacesList() {
           </Typography>
         }
         buttons={
-          <>
-            <StyledButton
-              onClick={() => {
-                // eslint-disable-next-line no-alert
-                alert('TODO: Support deletion of multiple datasets');
-              }}
+          <Stack direction="row" gap={1}>
+            <WorkspaceButton
+              onClick={handleDeleteSelected}
+              disabled={selectedItems.size === 0 || isDeleting.current}
+              tooltip="Delete selected workspaces"
             >
-              <DeleteIcon color="primary" />
-            </StyledButton>
-            <CreateWorkspaceButton handleCreateWorkspace={handleCreateWorkspace} />
-          </>
+              <DeleteRounded />
+            </WorkspaceButton>
+            <WorkspaceButton onClick={handleCreateWorkspace} tooltip="Create workspace">
```

Comment:
Is this just a placeholder until the new dialog is available? `handleCreate` workspace expects a `workspaceName` arg,

Reply:
This is just a placeholder for now, yes; once your PR is in, I'll update this to launch the dialog instead.